src/webhooks/index.ts

```diff
@@ -1,8 +1,10 @@
-import {WebhooksRegistry} from './registry';
+import {ConfigInterface} from '../base-types';
 
-const ShopifyWebhooks = {
-  Registry: WebhooksRegistry,
-};
+import {gdprTopics, webhooksRegistry} from './registry';
```

Comment:
I wonder if we should export the GDPR topics as an enum out of the main export, with the other types?